pyproject.toml

```diff
@@ -51,3 +52,7 @@ default_section = "THIRDPARTY"
 [tool.pytest.ini_options]
 
 [tool.setuptools_scm]
+local_scheme = "no-local-version"
+
+[tool.setuptools.packages.find]
```

Comment:
Because of our flat package structure (`dev` and `timely_beliefs` are both found by default), we need to give setuptools a hint to either explicitly exclude the `dev` package, or to explicitly include the `timely_beliefs` package.